src/app/cars/cars.component.html

```diff
@@ -1 +1,398 @@
-<p>cars works!</p>
+<div id="carsPage">
+  <div class="car">
```

Comment:
The design of this component can improved. This page looks like just 4 of the same sections repeated over and over:
```
header
image
description
hide/show spec button
spec table
```
Not sure if it is in the scope of this PR, but my suggestion would be to create one `car-info.component` that has that template, and accept different arguments for the different car information (perhaps given as a model). then in this `car.component`, it would just have an array of 5 cars, and run an `ngFor` to display the car info.

What do you think @JaydenBischoff , later issue or now?

Reply:
I think that it would be good to work on this now and add it to this pr